pysatCDAAC/instruments/methods/general.py

```diff
@@ -96,14 +99,20 @@ def download(date_array, tag, inst_id, supported_tags=None,
         try:
             # Uncompress files and remove tarball
             tar = tarfile.open(fname)
-            tar.extractall(path=os.path.join(data_path, yrdoystr))
+            if sub_path:
+                # Send to subdirectory.
+                tar.extractall(path=os.path.join(data_path, yrdoystr))
```

Comment:
I did find something seconds after posting the last review. My files ended up under `data_dir/cosmic2/ivm`. I didn't get the sub directories based upon yrdoy as expected.

Reply:
Looks like I needed to add `sub_path=True` to the file.  Update pending...

Reply:
Actually, do we need that?  The other ivm instruments don't have subdirectories for year/day, and we only have one file per instrument per day. Plus, the list_files function would need to be updated, etc.

Reply:
Looks like the `sub_path` option was specifically added so that the two would behave differently...

Reply:
I thought I saw sub_path=True in the relevant call which is why I brought up the sub directories. Now I don't see it..... I thought I was reporting a bug 🤷‍♂️

Reply:
It's partly my fault for not remembering what I wrote a year ago. 😕